openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -570,7 +570,7 @@ def get_all_courses(cls, orgs=None, filter_=None):
         # Note: If a newly created course is not returned in this QueryList,
         # make sure the "publish" signal was emitted when the course was
         # created. For tests using CourseFactory, use emit_signals=True.
-        course_overviews = CourseOverview.objects.all()
+        course_overviews = CourseOverview.all()
```

Comment:
Can you explain what's happening that's different because you're not referring to `object.all()`?

Reply:
I broke the code... I made a change w/o running the tests before I pushed this version up. =)